genesis/src/main.rs

```diff
@@ -135,21 +139,21 @@ fn create_starbase(nats: &Client, params: &UniverseParameters) -> Result<(), Box
             "decs.components.{}.{}.position",
             params.shard_name, entity_id
         ),
-        json!({
+        json!({"params": {
```

Comment:
Does it make more sense to accept raw JSON input to the `create_component` function, and then have the create_component function wrap that value inside a `params` field?

Reply:
This might be easier, that would change it so the only place we worry about the "params" field is in create_component... I can make this change